articles/quickstart/spa/_includes/_calling_api_use_rules.md

```diff
@@ -0,0 +1,7 @@
+::: panel Checkpoint
+Try to log in to your application again. Look at how the `access_token` differs from before. It is no longer an opaque token. Instead, it is now a JSON Web Token with a payload that contains your API identifier as audience and the scopes you created.
```

Comment:
I'd add a link either to https://auth0.com/docs/tokens/access-token or to https://auth0.com/docs/jwt